osmosis-1/osmosis-1.assetlist.json

```diff
@@ -5083,6 +5083,42 @@
         "OSMO": 857
       }
     },
+    {
+      "description": "The native EVM, governance and staking token of the Imversed",
+      "denom_units": [
+        {
+          "denom": "ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4",
+          "exponent": 0,
+          "aliases": [
+            "aimv"
+          ]
+        },
+        {
+          "denom": "imv",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4",
+      "name": "Imversed",
+      "display": "imversed",
+      "symbol": "IMV",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-517",
+        "source_denom": "aimv"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/imversed/images/imversed.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/imversed/images/imversed.svg"
+      },
+      "coingecko_id": "imv",
+      "keywords": [
+        "osmosis-frontier"
+      ],
+      "pools": {
+        "USDC.axl": 864
```

Comment:
Not a problem, but did we mean to have a 3% fee pool? Not against any rules, but unusual is all(, when most pools have somewhere around 0.3%...)

Reply:
It's strange, I think I set 0.3% fee in json config. This is json for pool creation: 
```
{
  "weights": "5ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4,5ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858",
  "initial-deposit": "50ibc/92B223EBFA74DB99BEA92B23DEAA6050734FEEAABB84689CB8E1AE8F9C9F9AF4,50ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858",
  "swap-fee": "0.03",
  "exit-fee": "0",
  "future-governor": ""
}
```

Reply:
Thanks for showing the config. 0.03 would equal 3%, while 0.003 would be 0.3%. Sorry about that